.ai-agents/memory-bank/activeContext.md

```diff
@@ -98,8 +95,7 @@ The current focus is on enhancing the Reviewer User experience with AI-driven an
 ## Active Decisions and Considerations
```

Comment:
It is not this line, it appeared that the explanation of `prisma` remained on line 142 of this file.
```
   - Ensure DATABASE_URL environment variable is properly set for prismaExtension
```

Reply:
Oh! I'll delete it!

Reply:
Fix it: https://github.com/liam-hq/liam/pull/1223/commits/a77afb77a5b4630752f6e57878567612e1e9a73f